src/types/Todo.ts

```diff
@@ -1,4 +1,5 @@
 export interface Todo {
+  currentTodoReducer: any;
```

Comment:
shouldn't be here - remove it

Reply:
<img width="573" alt="image" src="https://github.com/user-attachments/assets/77aefc3c-7e51-4bdc-8526-28f4d8511a47">